Quotes around Python API Exception Arguments

wbinventor opened the pull request.

paulromano commented on openmc/filter.py:
```diff
@@ -109,7 +109,7 @@ def type(self, type):
             self._type = type
         elif type not in FILTER_TYPES.values():
-            msg = 'Unable to set Filter type to "{0}" since it is not one ' \
+            msg = 'Unable to set Filter type to ""{0}" since it is not one ' \
```
Mistake here?

wbinventor replied:
Fixed
On Aug 4, 2015 9:32 PM, "Paul Romano" <notifications@github.com> wrote:

> In openmc/filter.py
> <https://github.com/mit-crpg/openmc/pull/428#discussion_r36268654>:
>
> > @@ -109,7 +109,7 @@ def type(self, type):
> >          if type is None:
> >              self._type = type
> >          elif type not in FILTER_TYPES.values():
> > -            msg = 'Unable to set Filter type to "{0}" since it is not one ' \
> > +            msg = 'Unable to set Filter type to ""{0}" since it is not one ' \
>
> Mistake here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/openmc/pull/428/files#r36268654>.
>

paulromano commented on openmc/material.py:
```diff
@@ -332,13 +332,13 @@ def get_all_nuclides(self):
 
         string += '{0: <16}{1}{2}'.format('\tDensity', '=\t', self._density)
-        string += ' [{0}]\n'.format(self._density_units)
+        string += ' ["{0}"]\n'.format(self._density_units)
```
Aren't quotes a little redundant in this case?

wbinventor replied:
Definitely agree, removed them. The search-replace macros I used for this got a little out of hand in a few places evidently :-)

paulromano commented on openmc/tallies.py:
```diff
@@ -618,23 +618,23 @@ def get_tally_xml(self):
+                    nuclides += '"{0}" '.format(nuclide.name)
                 else:
-                    nuclides += '{0} '.format(nuclide)
+                    nuclides += '"{0}" '.format(nuclide)
```
Won't this break tally input with nuclides?

paulromano commented on openmc/tallies.py:
```diff
@@ -618,23 +618,23 @@ def get_tally_xml(self):
             scores = ''
             for score in self.scores:
-                scores += '{0} '.format(score)
+                scores += '"{0}" '.format(score)
```
Same thing here but for scores

paulromano commented on openmc/tallies.py:
```diff
@@ -609,7 +609,7 @@ def get_tally_xml(self):
                 bins = ''
                 for bin in filter.bins:
-                    bins += '{0} '.format(bin)
+                    bins += '"{0}" '.format(bin)
```
and here

paulromano commented on openmc/trigger.py:
```diff
@@ -92,7 +92,7 @@ def add_score(self, score):
 
         if not isinstance(score, basestring):
-            msg = 'Unable to add score "{0}" to tally trigger since ' \
+            msg = 'Unable to add score ""{0}"" to tally trigger since ' \
```
double quotes again

paulromano commented on openmc/universe.py:
```diff
@@ -343,7 +343,7 @@ def create_xml_subelement(self, xml_element):
             for surface_id in self._surfaces:
                 # Determine if XML element already includes this Surface
-                path = './surface[@id=\'{0}\']'.format(surface_id)
+                path = './surface[@id=\'"{0}"\']'.format(surface_id)
```
I suspect this might not be wanted

paulromano commented on openmc/universe.py:
```diff
@@ -355,7 +355,7 @@ def create_xml_subelement(self, xml_element):
                 # Append the halfspace and Surface ID
                 halfspace = self._surfaces[surface_id][1]
-                surfaces += '{0} '.format(halfspace * surface_id)
+                surfaces += '"{0}" '.format(halfspace * surface_id)
```
probably will break input

paulromano commented on openmc/universe.py:
```diff
@@ -1149,19 +1149,19 @@ def universes(self, universes):
+        string += '{0: <16}"{1}""{2}"\n'.format('\tName', '=\t', self._name)
+        string += '{0: <16}"{1}""{2}"\n'.format('\t# Rings', '=\t', self._num_rings)
+        string += '{0: <16}"{1}""{2}"\n'.format('\t# Axial', '=\t', self._num_axial)
+        string += '{0: <16}"{1}""{2}"\n'.format('\tCenter', '=\t',
```
All of these will probably look weird

paulromano commented on openmc/universe.py:
```diff
@@ -1197,7 +1197,7 @@ def create_xml_subelement(self, xml_element):
         if self._outer is not None:
             outer = ET.SubElement(lattice_subelement, "outer")
-            outer.text = '{0}'.format(self._outer._id)
+            outer.text = '"{0}"'.format(self._outer._id)
```
probably will break input

paulromano commented:
Ok, I think I commented on all the obvious errors. Once you're done addressing my comments, can you just take a careful look through the diffs and see if anything else looks awry?

wbinventor commented:
@paulromano sorry that this was so rough around the edges for the initial review. I just addressed all of the issues you spotted, and reviewed the entire thing myself, and believe it is good to go.

paulromano commented:
No worries -- that's what reviews are for!
